### ISSUE TITLE
Token is not generating 

### ISSUE THREAD

**Issue by goppi1** [zxol/airbnbapi#42]
Hi,

I am trying to get token for airbnb and it returning null in response. Here is the code I am using
`airbnb.login({ username: 'xxxxx', password: 'xxxxx }).then(function(data) {
    console.log('data: ', data);
  })`

It returning null in console. 

`airbnb.newAccessToken({ username: 'xxxxxx', password: 'xxxxxx' }).then(function(data) {
  console.log('data: ', data);
})`

Above code returning error:- 
`data:  {
  error: {
    error_code: 420,
    error: 'unknown_error',
    error_message: 'Unable to perform action. Please try again through the website or contact support if you need immediate assistance.'
  }
}`

Please help me and let me know where I am doing wrong.

**Comment by wazcov** [comment]
I'm also getting this with a new airbnb account

**Comment by igorzelaya** [comment]
Same here.  Maybe its because my user is linked with my gmail account?  Its yours also linked?

**Comment by bobmarley29** [comment]
i've started using this lib https://github.com/drawrowfly/airbnb-private-api . its does not have many endpoint support as this one but it works great at least for the token part

**Comment by itbankas** [comment]
Yes there is another way to get token via v2/authentications a bit different json post
old one /logins  (often goes down)
body: {
                    authenticationParams: {
                        ...(type === 'email'
                            ? {
                                  email: {
                                      email: this.email,
                                      password: this.password,
                                  },
                              }
                            : {
                                  phone: {
                                      deliveryMethod: 'TEXT',
                                      isCombinedFlow: false,
                                      isGlobal: true,
                                      otp: code,
                                      phoneNumber: this.phone,
                                  },
                              }),
                    },
                    metadata: {
                        sxsMode: 'OFF',
                    },
                },
